File: 9-palindrome-number/9-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        strX = str(x)                             
        length = len(strX)                    
        for i in range(length // 2):             
            if strX[i] != strX[length - 1 - i]: 
                return False
        return True

File: 9-palindrome-number/9-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string s = to_string(x);
        int n = s.size();
        for(int i=0; i<n; i++) {
            //cout << s[0] << " " << s[n-i-1] << endl;
            if(s[i] != s[n-i-1])
                return false;
        }
        return true;
    }
};